Steer/DigitizerWorkflow/src/SimpleDigitizerWorkflow.cxx

```diff
@@ -93,19 +94,7 @@ void customize(std::vector<o2::framework::CompletionPolicy>& policies)
 {
   using o2::framework::CompletionPolicy;
   // we customize the completion policy for the writer since it should stream immediately
-  auto matcher = [](DeviceSpec const& device) {
-    bool matched = device.name == "TPCDigitWriter";
-    if (matched) {
-      LOG(INFO) << "DPL completion policy for " << device.name << " customized";
-    }
-    return matched;
-  };
-
-  auto policy = [](gsl::span<o2::framework::PartRef const> const& inputs) {
-    return CompletionPolicy::CompletionOp::Consume;
-  };
-
-  policies.push_back({CompletionPolicy{"process-any", matcher, policy}});
+  policies.push_back(CompletionPolicyHelpers::defineByName("TPCDigitWriter", CompletionPolicy::CompletionOp::Consume));
```

Comment:
@sawenzel the helper `defineByName` does the same logic: whenever the device `TPCDigitWriter` is matched, `CompletionOp::Consume` is returned regardless of the actual message content. The only thing which is skipped is the log message. We could consider adding the log message to the helpers as well.